include/rdma/fabric.h

```diff
@@ -544,6 +544,7 @@ enum fi_type {
 	FI_TYPE_CQ_EVENT_FLAGS,
 	FI_TYPE_MR_MODE,
 	FI_TYPE_OP_TYPE,
+	FI_TYPE_NAME,
```

Comment:
Yes, agree. I just wanted to add behavior like toString() method from Java, C# that can be used for string too.
I'll remove this